lib/neat.rb

```diff
@@ -1,12 +1,14 @@
-require "sass"
 require "neat/generator"
 
 module Neat
-  if defined?(Rails::Engine)
+  if defined?(Rails) && defined?(Rails::Engine)
     class Engine < ::Rails::Engine
       config.assets.paths << File.expand_path("../core", __dir__)
     end
   else
-    Sass.load_paths << File.expand_path("../core", __dir__)
+    begin
+      require "sass"
+      Sass.load_paths << File.expand_path("../core", __dir__)
+    rescue LoadError; end
```

Comment:
Nitpick: I'd move `end` to its own line.

Reply:
You know I love a good nitpick 😄 

Fixed in a9202bfd5b8faad993e356342b010da00c751983